.gitignore

```diff
@@ -3,6 +3,7 @@
 *.dll
 *.so
 *.dylib
+*.vscode
```

Comment:
This change does not seem to actually be anywhere in basicnet. Can you add it to the ping PR and remove it from here?

For future reference, I usually sync via something like this (actual command untested)

`cd $GOPATH/src/github/livepeer/go-livepeer-basicnet && git archive HEAD | tar -C $GOPATH/src/github/livepeer/go-livepeer/vendor/github/livepeer/go-livepeer-basicnet -xf -`

Then sanity check via `git status`, `git diff` etc. That should prevent extraneous changes from creeping in the commit.

Reply:
Yeah - this is added for the `go-livepeer` repo instead of the `go-livepeer-basicnet` repo.